<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Farm">
       <attribute name="title">
        <string>Betrieb</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lab_FarmCaption">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Betriebe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tv_Farms"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_FarmsSelectMarked">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Markierten Eintrag auswählen</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="fl_Farm">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lab_FarmKeyword">
            <property name="text">
             <string>Begriff:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ed_FarmName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lab_concerning">
            <property name="text">
             <string>enthalten in ...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="cB_FarmerName">
            <property name="text">
             <string>Landwirtsnamen</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="cB_FarmDescription">
            <property name="text">
             <string>Beschreibung</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cB_FarmName">
            <property name="text">
             <string>Betriebsnamen</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_FarmsSearchAndMark">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Suchen und Markieren</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Cows">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Kuhtabelle</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lab_CowsDescription">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Kühe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tv_Cows"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_SelectMarked">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Markierten Eintrag auswählen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="fl_Cows">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="f_CowsLeft">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <property name="rowWrapPolicy">
               <enum>QFormLayout::DontWrapRows</enum>
              </property>
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <property name="verticalSpacing">
               <number>3</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item row="1" column="0">
               <widget class="QCheckBox" name="cb_SearchCowByName">
                <property name="text">
                 <string>Nach Kuhnamen suchen:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="ed_CowName"/>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="cb_MinFatProteinQuotient">
                <property name="text">
                 <string>Fett-Eiweiß-Quotient unter</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="sb_MinFatProteinQuotient">
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="singleStep">
                 <double>0.200000000000000</double>
                </property>
                <property name="value">
                 <double>1.200000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="cb_MaxFatProteinQuotient">
                <property name="text">
                 <string>Fett-Eiweiß-Quotient über</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="sb_MaxFatProteinQuotient">
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="singleStep">
                 <double>0.200000000000000</double>
                </property>
                <property name="value">
                 <double>1.400000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="F_CowsRight">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="2" column="0">
               <widget class="QCheckBox" name="cb_MinCellCount">
                <property name="text">
                 <string>Zellzahlen über</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="cb_FatPercentage">
                <property name="text">
                 <string>Fettwert (in %) unter </string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="sb_FatPercentage">
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.200000000000000</double>
                </property>
                <property name="value">
                 <double>3.700000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="cb_MaxCellCount">
                <property name="text">
                 <string>Zellzahlen unter</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="sb_MinCellCount">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>2000000</number>
                </property>
                <property name="singleStep">
                 <number>50000</number>
                </property>
                <property name="value">
                 <number>200000</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="sb_MaxCellCount">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>2000000</number>
                </property>
                <property name="singleStep">
                 <number>50000</number>
                </property>
                <property name="value">
                 <number>800000</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_CowsSearchAndMark">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Suchen und Markieren</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_CowView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Kuhübersicht</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lab_CowViewDescription">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Eigenschaften der Kuh ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tv_CowView"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_SaveChanges">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Änderungen speichern</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Diagram">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Diagramm</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGraphicsView" name="gv_Diagram"/>
        </item>
        <item>
         <layout class="QFormLayout" name="fl_diagram">
          <item row="1" column="0">
           <widget class="QLabel" name="lab_XAxis">
            <property name="text">
             <string>X-Achse</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lab_YAxis">
            <property name="text">
             <string>Y-Achse</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cob_XAxis">
            <item>
             <property name="text">
              <string>&lt;Der konstante Anstieg, z.B. Zeit in Monaten&gt;</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cob_YAxis">
            <item>
             <property name="text">
              <string>&lt;Der dynamische Anstieg, z.B. Milchabgabe&gt;</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cb_diagramForSingleCo">
            <property name="text">
             <string>Diagramm über eine Einzelne Kuh?</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cob_CowNrName">
            <item>
             <property name="text">
              <string>&lt;Nummer/Name der Kuh&gt;</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_CreatDiagram">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Diagramm erstellen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Report">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Auswertung</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGraphicsView" name="gv_Report"/>
        </item>
        <item>
         <layout class="QFormLayout" name="fl_ReportProperties">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="1">
           <widget class="QLabel" name="lab_Property">
            <property name="text">
             <string>Eigenschaft</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cob_Property"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lab_FromTable">
            <property name="text">
             <string>aus Tabelle ...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cob_FromTable"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_AddProperty">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Eigenschaft hinzufügen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <widget class="QMenu" name="menuHow">
     <property name="title">
      <string>Wie ...</string>
     </property>
     <addaction name="action_ToChoseAFarm"/>
     <addaction name="action_ToCheckCowDetails"/>
     <addaction name="action_ToCreateDiagrams"/>
     <addaction name="action_ToCreateReports"/>
    </widget>
    <widget class="QMenu" name="menuWhy">
     <property name="title">
      <string>Warum ...</string>
     </property>
     <addaction name="action_AreTabsDisabled"/>
    </widget>
    <widget class="QMenu" name="menuWhere">
     <property name="title">
      <string>Wo ...</string>
     </property>
     <addaction name="action_ToPrint"/>
     <addaction name="action_ToReadNewData"/>
     <addaction name="action_ToSaveMyTemplates"/>
    </widget>
    <addaction name="menuHow"/>
    <addaction name="menuWhy"/>
    <addaction name="menuWhere"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionTeam"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Auswertung</string>
    </property>
    <widget class="QMenu" name="menuRecentlyUsedReports">
     <property name="title">
      <string>Kürzlich verwendete Vorlagen</string>
     </property>
     <addaction name="action_no_Reports"/>
    </widget>
    <widget class="QMenu" name="menuExportReport">
     <property name="title">
      <string>Exportieren oder Drucken</string>
     </property>
     <widget class="QMenu" name="menuExportReportAs">
      <property name="title">
       <string>Exportieren als ...</string>
      </property>
      <addaction name="action_Report_TextFile"/>
      <addaction name="action_Report_PDF"/>
      <addaction name="action_Report_DOC"/>
     </widget>
     <addaction name="actionPrintReport"/>
     <addaction name="menuExportReportAs"/>
    </widget>
    <addaction name="menuRecentlyUsedReports"/>
    <addaction name="separator"/>
    <addaction name="actionNewReportTemplate"/>
    <addaction name="actionLoadReportTemplate"/>
    <addaction name="actionSaveReportTemplate"/>
    <addaction name="separator"/>
    <addaction name="menuExportReport"/>
   </widget>
   <widget class="QMenu" name="menuDiagram">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Diagramm</string>
    </property>
    <widget class="QMenu" name="menuRecentlyUsedDiagrams">
     <property name="title">
      <string>Kürzlich verwendete Vorlagen</string>
     </property>
     <addaction name="action_no_Diagrams"/>
    </widget>
    <widget class="QMenu" name="menuExportDiagram">
     <property name="title">
      <string>Exportieren oder Drucken</string>
     </property>
     <widget class="QMenu" name="menuExportDiagramAs">
      <property name="title">
       <string>Exportieren als ...</string>
      </property>
      <addaction name="action_Diagram_JPEG"/>
      <addaction name="action_Diagram_SVG"/>
      <addaction name="action_Diagram_PDF"/>
     </widget>
     <addaction name="actionPrintDiagram"/>
     <addaction name="menuExportDiagramAs"/>
    </widget>
    <addaction name="menuRecentlyUsedDiagrams"/>
    <addaction name="separator"/>
    <addaction name="actionNewDiagramTemplate"/>
    <addaction name="actionSaveDiagramTemplate"/>
    <addaction name="actionLoadDiagramTemplate"/>
    <addaction name="separator"/>
    <addaction name="menuExportDiagram"/>
   </widget>
   <widget class="QMenu" name="menuDataSet">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Datensätze</string>
    </property>
    <addaction name="actionReadFromFile"/>
   </widget>
   <addaction name="menuData"/>
   <addaction name="menuDataSet"/>
   <addaction name="menuDiagram"/>
   <addaction name="menuReport"/>
   <addaction name="menuHelp"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Zurücksetzen</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Über</string>
   </property>
  </action>
  <action name="actionTeam">
   <property name="text">
    <string>Team</string>
   </property>
  </action>
  <action name="action_ToChoseAFarm">
   <property name="text">
    <string>... wähle ich einen Betrieb aus?</string>
   </property>
  </action>
  <action name="action_ToCheckCowDetails">
   <property name="text">
    <string>... betrachte ich eine Kuh genauer?</string>
   </property>
  </action>
  <action name="action_ToCreateDiagrams">
   <property name="text">
    <string>... erstelle ich ein Diagramm?</string>
   </property>
  </action>
  <action name="action_ToCreateReports">
   <property name="text">
    <string>... fertige ich eine Auswertung an?</string>
   </property>
  </action>
  <action name="action_AreTabsDisabled">
   <property name="text">
    <string>... sind die Reiter anfangs ausgegraut?</string>
   </property>
  </action>
  <action name="action_ToPrint">
   <property name="text">
    <string>... drucke ich?</string>
   </property>
  </action>
  <action name="action_ToReadNewData">
   <property name="text">
    <string>... lese ich neue Messergebnisse ein?</string>
   </property>
  </action>
  <action name="action_ToSaveMyTemplates">
   <property name="text">
    <string>... sichere ich meine Vorlagen?</string>
   </property>
  </action>
  <action name="actionLoadReportTemplate">
   <property name="text">
    <string>Vorlage laden</string>
   </property>
  </action>
  <action name="actionSaveReportTemplate">
   <property name="text">
    <string>Vorlage speichern</string>
   </property>
  </action>
  <action name="actionNewReportTemplate">
   <property name="text">
    <string>Neue Vorlage</string>
   </property>
  </action>
  <action name="action_no_Reports">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>(keine)</string>
   </property>
  </action>
  <action name="actionNewDiagramTemplate">
   <property name="text">
    <string>Neue Vorlage</string>
   </property>
  </action>
  <action name="actionSaveDiagramTemplate">
   <property name="text">
    <string>Vorlage speichern</string>
   </property>
  </action>
  <action name="actionLoadDiagramTemplate">
   <property name="text">
    <string>Vorlage laden</string>
   </property>
  </action>
  <action name="action_no_Diagrams">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>(keine)</string>
   </property>
  </action>
  <action name="actionPrintDiagram">
   <property name="text">
    <string>Drucken</string>
   </property>
  </action>
  <action name="action_Diagram_JPEG">
   <property name="text">
    <string>... Bild (JPEG)</string>
   </property>
  </action>
  <action name="action_Diagram_SVG">
   <property name="text">
    <string>... Vektorgrafik (SVG)</string>
   </property>
  </action>
  <action name="action_Diagram_PDF">
   <property name="text">
    <string>... PDF</string>
   </property>
  </action>
  <action name="actionPrintReport">
   <property name="text">
    <string>Drucken</string>
   </property>
  </action>
  <action name="action_Report_TextFile">
   <property name="text">
    <string>... Textdatei</string>
   </property>
  </action>
  <action name="action_Report_PDF">
   <property name="text">
    <string>... PDF</string>
   </property>
  </action>
  <action name="action_Report_DOC">
   <property name="text">
    <string>... Worddokument</string>
   </property>
  </action>
  <action name="actionReadFromFile">
   <property name="text">
    <string>Aus Datei einlesen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
